Readme.md

```diff
@@ -2,9 +2,9 @@
 
 Tsserver language server extension for [coc.nvim](https://github.com/neoclide/coc.nvim).
 
-Most code from `typescript-language-features` extension which bundled with VSCode.
+Most of the code is from `typescript-language-features` extension which is bundled with VSCode.
 
-**Note:** if you're using nvm, you need configure `tsserver.npm` to your global
```

Comment:
It's https://github.com/creationix/nvm not neovim.

Reply:
Ahh! Sorry that's my mistake. I read it a bit too quickly and assumed you were mentioning neovim and not the node version manager.